src/users/users.service.ts

```diff
@@ -53,4 +54,19 @@ export class UsersService {
       throw error;
     }
   }
+
+  async updatePhoneNumber(id: number, user: UpdatePhoneNumberDto) {
+    try {
+      return this.prismaService.user.update({
```

Comment:
lack of await, try catch would not work. 
Also think about the error itself, you are attempting to catch it but you throw it immidietly after catching. Please go back to the lesson about error handling and consider what Prisma error code may occur here. Throw proper error based on the error code.